components/brave_ads/common/search_result_ad_util_unittest.cc

```diff
@@ -0,0 +1,133 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_ads/common/search_result_ad_util.h"
+#include "base/strings/string_piece.h"
+#include "services/network/public/cpp/resource_request.h"
+#include "testing/gtest/include/gtest/gtest.h"
+#include "url/gurl.h"
+
+namespace brave_ads {
+
+namespace {
```

Comment:
bad practice. tests should not be in anonymous namespace.

Reply:
removed anonymous namespace